routes.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Do you really need any of these configurations? 🤔  
I reckon that all your needs are covered by implicitly created api from `db.json` file.